README.md

```diff
@@ -74,6 +74,9 @@ See [CONTRIBUTING.md](docs/CONTRIBUTING.md).
 
 ## 🖌 Userstyles
 
+> [!NOTE]
+> Userstyles labeled with the 🚧 emoji are looking for maintainers. Feel free to contribute or request to maintain any you are passionate about!
```

Comment:
Thoughts? I think its worthy of the important admonishment, and I like being verbose in the three areas:

1. Describing what the emoji is.
2. Contributions are welcome.
3. A simple pointer to where they can add themselves as maintainer. (Could be better moving this out to a separate section in the document to link towards? I'm not too fussed about the verbosity of this message in the README though)

```suggestion
> [!IMPORTANT]
> Userstyles labeled with the "🚧" emoji are looking for maintainers, and may not work as intended. Contributions are still welcome and encouraged. If you would like to become a maintainer, add your username to the `current-maintainers` array in the [userstyles.yml](./scripts/userstyles.yml) file.
```

Reply:
Yeah I like that. We should link to https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md#userstylesyml though since there are more steps than just adding your username to the array.

Reply:
Yeah I was thinking that but I'd prefer those docs to be rewritten so that there's an explicit section on "adding yourself as a maintainer"

At most its two steps right?

1. add your name into the collaborators array
2. find the userstyle(s) and add your username to the current-maintainers array?

Reply:
I have some WIP docs over at https://github.com/uncenter/ctp-userstyles/blob/docs-quick-tips/docs/quick-tips.md that I've been slowly working on. Might be nice to make a larger PR rewriting parts of the docs and working those tips in, and those changes you just mentioned. We can track these at #449? Feel free to add a comment or edit the todo list if you want.

Reply:
Sure, I'll get around to that at some point. 

Are you happy if I commit the suggestion and merge this PR for now? We can link out to more detailed instructions once a docs PR lands.

Reply:
Sounds good!